tools/base/tests/test_runner.py

```diff
@@ -169,6 +175,103 @@ class DummyRecord(object):
         assert not logfilter.filter(DummyRecord())
 
 
+# BazelAdapter tests
+
+def test_bazeladapter_constructor():
+    _runner = DummyRunner()
```

Comment:
Why do local variables have `_` prefix?

Reply:
my use is a bit inconsistent i guess - its used here to separate it from the `runner` module

in terms of why `_` in particular - googling "python local variable underscore prefix" responds with:

> The underscore prefix is meant as a hint to another programmer that a variable or method starting with a single underscore is intended for internal use. This convention is defined in PEP 8. This isn't enforced by Python. Python does not have strong distinctions between “private” and “public” variables like Java does.

Reply:
lmk if you want this changed and i will give these vars a different name

Reply:
Isn't every local variable for internal use? :) Not sure if I'm missing something. I understand for class variables, but this is not that.

Reply:
hmm maybe yes/no - i think using in this way is not uncommon, but im gonna just change their names to avoid any confusion